guides/common/modules/proc_remediating-compliance-failures.adoc

```diff
@@ -10,24 +11,36 @@ Remediation might render the system non-functional.
 ====
 
 .Prerequisites
-* Your user account has a role assigned that has the `view_arf_reports` and `view_hosts` permissions.
+* Your user account has a role assigned that has the following permissions: `view_arf_reports`, `view_hosts`, `create_job_invocations`
 
 .Procedure
-To examine a simplified compliance policy report:
+To examine a simplified compliance policy report and run the compliance failure remediation wizard:
 
 . In the {ProjectWebUI}, navigate to *Hosts* > *Compliance* > *Reports*.
 . In the *Reported At* column, click the time link of the report you want to examine.
-{Project} displays a simplified list of policy rules with the results of the scan.
+{Project} displays a list of log messages describing the results of the scan.
+. Locate a log message that describes a failed compliance check.
+In the *Actions* column, select *Remediation* to open the compliance remediation wizard.
+Follow the wizard to remediate the compliance failure.
++
+[NOTE]
+====
+The remediation wizard might not be available for all compliance failures.
+====
 
-To examine a full compliance policy report and remediate compliance failures manually:
+To examine a full compliance policy report:
```

Comment:
This looks like two procedures in a single module. I recommend that either you create a separate module or try to combine the procedures, perhaps as subprocedures.

2. To view a single report ....
3. To view a full report...

or

3. Optional: To view a full report...

Reply:
What I _really_ want to do is to delete the second sub-procedure :upside_down_face: 

Yeah, I'll do it. Because:

1. The wizard is new and it's now the preferred way of remediation, the most user-friendly one.
2. It's not that hard to find the full compliance report in the web UI. It's right there under the button labeled "Full report".

There is no need for the second sub-procedure anymore.